dle/api/views.py

```diff
@@ -17,6 +17,7 @@
 def searchkit(request: HttpRequest) -> JsonResponse:
     """Core search API which gets proxied to Elasticsearch"""
     es = get_client()
+    print(request.body)
```

Comment:
Probably an accidental debugging print statement here, but I can remove it when I make some tweaks to the visuals after this gets merged.